app/views/application/cycle/edit/check.njk

```diff
@@ -19,18 +19,20 @@
         {{name}}
       </span>
       <h1 class="govuk-heading-xl">
-        Check and confirm change of cycle
+        Check and confirm deferral to next cycle
       </h1>
 
-      <p>The course you have currently offered is not available yet in the next cycle.</p>
+      <p>You are deferring this offer to the next cycle (2020 to 2021).</p>
```

Comment:
Aren't we telling them exactly what they already know here?

Reply:
The most crucial part of the guidance now has been removed.

Reply:
No, it's the second sentence :)

Reply:
Oh fab.

Reply:
Still think the first line is redundant